### PR TITLE
Re-disable pay with max RR for moonDAO only

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

1. Re-disable pay with max RR for moonDAO only
2. Add tooltip for sequoia edge case

## Screenshots or screen recordings

1. 

Screen Shot 2022-01-21 at 10 37 16 am

2. 

Screen Shot 2022-01-21 at 10 37 05 am

## Acceptance checklist
- [x] I have evaluated the [Approval Guidelines](../CONTRIBUTING.md#approval-guidelines) for this PR.
- [x] I have tested this PR in [all supported browsers](../CONTRIBUTING.md#browser-support).

### CODE REVIEW

File: src/components/Dashboard/Pay/index.tsx [hunk 1]
```diff
@@ -55,9 +55,14 @@ export default function Pay() {
     const payButtonText = metadata.payButton?.length
       ? metadata.payButton
       : t`Pay`
+
+    // v1 projects who still use 100% RR to disable pay
     const isV1AndMaxRR =
       terminal?.version === '1' && fcMetadata.reservedRate === 200
 
+    // Edge case for MoonDAO, upgraded to v1.1 but can't use payIsPaused for now
+    const isMoonAndMaxRR = projectId?.eq(199) && fcMetadata.reservedRate === 200
```
Comment (aeolianeth) [inline src/components/Dashboard/Pay/index.tsx:64]: **suggestion**: can we have `199` in a constant so it's less magic?

Comment (johnnyd-eth) [reply src/components/Dashboard/Pay/index.tsx]: Yeah good idea, got it

**Review by aeolianeth — APPROVED** [review]
Nice job, love the added constants file. Approving 🚀

File: src/constants/projectIds.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export const CONSTITUTIONDAO_PROJECT_ID = 36
```
Comment (peripheralist) [inline src/constants/projectIds.ts:1]: thanks for cleaning up this system 😁 just a nit pick, but wdyt about making this an object like this?

```
PROJECT_IDS = {
  CONSTITUTION_DAO: 36
}
```

File: src/components/Dashboard/Pay/index.tsx [hunk 3]
```diff
@@ -55,9 +56,15 @@ export default function Pay() {
     const payButtonText = metadata.payButton?.length
       ? metadata.payButton
       : t`Pay`
+
+    // v1 projects who still use 100% RR to disable pay
     const isV1AndMaxRR =
       terminal?.version === '1' && fcMetadata.reservedRate === 200
 
+    // Edge case for MoonDAO, upgraded to v1.1 but can't use payIsPaused for now
```
Comment (peripheralist) [inline src/components/Dashboard/Pay/index.tsx:64]: comments are solid 🤘

File: src/constants/projectIds.ts (hunk 2 above)
Comment (johnnyd-eth) [reply src/constants/projectIds.ts:1]: Love it, will do

**Pull request merged**